expirationd.lua

```diff
@@ -271,6 +271,9 @@ end
 --   options = {      -- (table with named options)
 --     * process_expired_tuple -- Applied to expired tuples, receives (space_id, args, tuple) as arguments;
 --                                can be nil: by default, tuples are removed.
+--     * index                 -- Name of the index to iterate on; if omitted, will use the primary index;
```

Comment:
Also it would be great to allow to pass `index_id`. It could be easily done if you fix my comment https://github.com/tarantool/expirationd/pull/52/commits/c596f8d73c3a71af2aa5139f7ca5d120eee9700d#r658927759

Just note that `create_task ` applies space_id not just space_name

Reply:
Yes, I forgot to change the description